### ISSUE TITLE
Can not update automatically!

### ISSUE THREAD

**Issue by ppa993** [NECROBOTIO/NecroBot#5346]
### Expected Behavior
Auto update new version successfully

### Actual Behavior
Ask for update appear then disappear real quick.
Even if I have a chance to say 'Y' and press Enter, it's start to download the update but then cancel the the updating process and continue to log in.

You can see it in below picture, the update file will disappear when the bot ignore the update and continue to log in.
![image](https://cloud.githubusercontent.com/assets/13492824/17613652/bf24118e-6089-11e6-9876-4ec0c95b8b97.png)

### Your config.json
"AutoUpdate": true

### Other Information
Version: 0.8.5
Windows Version: 7 x64 Enterprise

**Comment by Traika** [comment]
Same problem here

**Comment by nerclid** [comment]
Yes, had to clean install. Just finished, and it is up and running.

**Comment by Baleki** [comment]
problem after update

http://i.prntscr.com/e7c7ca3e5a224a9699dbbc6b926bd5fe.png

**Comment by g00tch** [comment]
Copy your config, manual download the new release, run setup, exit, paste your config, run.

https://github.com/NecronomiconCoding/NecroBot/releases/download/v0.8.6/Release_0_8_6.zip

**Comment by guiihjorge** [comment]
@nerclid how?

@Baleki vai brasil!

**Comment by guiihjorge** [comment]
@g00tch, i made it, now appears this

![image](https://cloud.githubusercontent.com/assets/20983908/17613878/fe962c0a-6037-11e6-8001-e3a6cf11c721.png)

**Comment by Baleki** [comment]
@g00tch nope, not work

**Comment by nerclid** [comment]
do not copy your config over from the old install, some of the variable names changed
you need to setup a clean install

**Comment by g00tch** [comment]
My 8.5 config is working perfectly fine with the 8.6.

**Comment by nerclid** [comment]
Mine did not. :(

**Comment by Baleki** [comment]
deleted all .json on config folder, run NecroConfig and notting

**Comment by g00tch** [comment]
I didn't bother with the beta NecroConfig. I'm just using the config.json

**Comment by Baleki** [comment]
Alson, 8.5 is running fine but only 1 accont and on human walking if get softban ip is ban too.

**Comment by zzxxzzxxSG** [comment]
Download v0.8.6 ZIP file.
Unzip.
Over ALL FILES over to your current Folder.

Start NecroBot.exe, it will prompt you to do something about Translation.
Key in "y" and enter.
It should work.

**Comment by Baleki** [comment]
notthing @zzxxzzxxSG

**Comment by guiihjorge** [comment]
@zzxxzzxxSG still not workin'

**Comment by zzxxzzxxSG** [comment]
Did you guys install/update to the new framework thing?
I'm on Windows10 and do not have issue.

**Comment by FantasyHuang** [comment]
Download v0.8.6 ZIP file.
Unzip.
Over ALL FILES over to your current Folder.
![crash](https://cloud.githubusercontent.com/assets/20943430/17614517/95b0797c-6099-11e6-84ae-02852fb98094.jpg)

Start NecroBot.exe,
And Then CRUSH!

**Comment by lkhgamer** [comment]
still using 085, wating for response about this new version
